boxsdk/object/base_object.py

```diff
@@ -208,7 +207,7 @@ def _paging_wrapper(self, url, starting_index, limit, factory=None):
             for index_in_current_page, item in enumerate(response['entries']):
                 instance_factory = factory
                 if not instance_factory:
-                    instance_factory = Translator().translate(item['type'])
+                    instance_factory = self._session.translator.translate(item['type'])
```

Comment:
Should `base_object` also define a `translator` property?

I'd prefer to refer to `self.translator` here and throughout the code, rather than to `self._session.translator`.